config/variables_test.go

```diff
@@ -8,54 +8,6 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-func TestFormatPrompt(t *testing.T) {
-	t.Parallel()
-
-	variable := Variable{Name: "foo"}
-	options := &BoilerplateOptions{}
-
-	actual := formatPrompt(variable, options)
-	expected := "Enter a value for variable 'foo'"
-
-	assert.Equal(t, expected, actual)
-}
-
-func TestFormatPromptWithDefault(t *testing.T) {
-	t.Parallel()
-
-	variable := Variable{Name: "foo", Default: "bar"}
-	options := &BoilerplateOptions{}
-
-	actual := formatPrompt(variable, options)
-	expected := "Enter a value for variable 'foo' (default: 'bar')"
-
-	assert.Equal(t, expected, actual)
-}
-
-func TestFormatPromptWithCustomPrompt(t *testing.T) {
-	t.Parallel()
-
-	variable := Variable{Name: "foo", Prompt: "custom"}
-	options := &BoilerplateOptions{}
-
-	actual := formatPrompt(variable, options)
-	expected := "custom"
-
-	assert.Equal(t, expected, actual)
-}
-
-func TestFormatPromptWithCustomPromptAndDefault(t *testing.T) {
-	t.Parallel()
-
-	variable := Variable{Name: "foo", Prompt: "custom", Default: "bar"}
-	options := &BoilerplateOptions{}
-
-	actual := formatPrompt(variable, options)
-	expected := "custom (default: 'bar')"
-
-	assert.Equal(t, expected, actual)
-}
-
```

Comment:
To be honest, these were not very useful tests. They are checking a part of the UI, and whenever you do that, you run a few risks:

1. UIs change often, so you spend a lot of time updating test code.
1. UI code, by its very nature, is designed for human consumption, and not computer consumption. This makes it hard to check the output. In particular, adding terminal colors means inserting all sorts of funky strings into the output, which makes the test code harder to write.
1. Most importantly, UI tests can only capture whether the right data is presented. They cannot typically capture if it "looks good". 

Obviously, this whole thing is a judgment call, but I felt that the tests that I deleted here were providing minimal value, were already expensive to maintain and were unlikely to catch any real bugs. When faced with the option of making the tests even more expensive by being able to handle colors, or refactoring the UI code significantly to support this use case--which may go away at the next re-design--or just removing them, I chose the latter.